modules/workbench/providers.tf

```diff
@@ -13,17 +13,17 @@ terraform {
 # values that cannot be determined until apply.`
 # " https://github.com/hashicorp/terraform/issues/26211#issuecomment-705084351
 # Uncomment to replace provider block when performing terraform import.
-//provider "google" {
-//  version = "3.5.0"
-//  project = "all-of-us-workbench-test"
-//  region  = "us-central1"
-//  zone    = "us-central1-c"
-//}
-
 provider "google" {
   version = "3.5.0"
-  project = var.project_id
-  region  = var.region
-  zone    = var.zone
+  project = "all-of-us-workbench-test"
```

Comment:
Was this for testing? Revert?

Reply:
You need it for `terraform import` to work in 0.13. Otherwise it tells you it doesn't know how to find the provider. Hopefully it can be removed soon.

Reply:
I don't follow, how does this get overridden when we use this module for an environment that is not all-of-us-workbench-test ? If it's not getting overridden, and the module still works properly for other projects, what is this parameter actually doing then?

Reply:
Good point. I think the right way to do it is to specify the provider only at the highest level.